.travis.yml

```diff
@@ -15,6 +15,5 @@ notifications:
     on_failure: never
 
 script:
```

Comment:
(I'm putting that here 'cause GitHub doesn't let me go higher)
At line 6, you should replace `10.13 by `lts/*` I think, it's even more future proof ;)
However, we might want to stay on 10 too, I let you judge.

Also, it could be nice to add some nice bells and whistle like Snyk and Greenkeeper to the project, to keep it safe (mostly) automatically.

Reply:
I think I'll leave the version at 10.13. Since the project is practically never built I don't want the next build to fail because a breaking change happened in Node